pymc3/distributions/continuous.py

```diff
@@ -4226,6 +4226,31 @@ class Interpolated(BoundedContinuous):
     ========  ===========================================
     Support   :math:`x \in [x\_points[0], x\_points[-1]]`
     ========  ===========================================
+    
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pymc3 as pm
+    import warnings
+    warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
This should not be used, if the code raises any warning we should address it instead of ignoring it. If you share the warning message we can help fix it.